go-flutter-desktop/hover
# Add AOT building

Using AOT engine builds from https://github.com/flutter-rs/engine-builds.
Also see https://github.com/go-flutter-desktop/go-flutter/issues/331
Test that AOT works:
- [ ] Darwin

1. Install hover AOT using `git clone https://github.com/jld3103/hover -b feature/aot`. Then go into `hover` and run `go install`.
2. Make sure `hover` is in your PATH. If not find a tutorial for your platform.
3. Run `hover build PLATFORM --release` where PLATFORM is your host OS (`linux`, `windows`, `darwin`)
4. Execute the resulting executable in `./go/build/outputs/PLATFORM/` (Preferrably in a terminal to catch any errors)
5. Report if it worked for you on your platform. Please also attach a zip of `./go/build/outputs/PLATFORM/` if something failed (building or running). Linux is already proven to work. Mac and Windows should work, but have to be tested.
